src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -689,6 +688,8 @@ public void StateChanged(long index)
             }
             catch (Exception e)
             {
+                if (_logger.IsInfoEnabled)
+                    _logger.Info($"Got exception during StateChanged({index}).");
```

Comment:
We don't need the actual exception, who needs to know what happened. Debugging such things is fun.